singularity-opac/base/indexers.hpp

```diff
@@ -31,6 +31,19 @@
 namespace singularity {
 namespace indexers {
 
+template <typename T>
+struct Identity {
+  Identity() = default;
+  Identity(T &&t) : data_(std::forward<T>(t)) {}
+
+  PORTABLE_INLINE_FUNCTION
+  Real &operator[](const int i) { return data_[i]; }
+  PORTABLE_INLINE_FUNCTION
+  Real &operator[](const int i) const { return data_[i]; }
+
+  T data_;
+};
+
```

Comment:
Is `Identity` used anywhere?

Reply:
Uh no. It could be deleted. It's essentially a silly test.